docker/Dockerfile

```diff
@@ -83,15 +91,17 @@ RUN set -e; \
     esac; \
     case "$DISTRO_ID-$DISTRO_VERSION_ID" in \
     'debian-'* | 'linuxmint-'* | 'ubuntu-'*) \
-        curl --retry "$DEB_MAX_ATTEMPT" --retry-connrefused --retry-delay 3 -fsSL 'https://bazel.build/bazel-release.pub.gpg' \
-        | gpg --dearmor \
-        | sudo tee /etc/apt/trusted.gpg.d/bazel.gpg \
-        >/dev/null; \
-        echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" \
-        | sudo tee /etc/apt/sources.list.d/bazel.list; \
-        sudo apt-get autoremove -y; \
-        sudo apt-get clean; \
-        sudo rm -rf /var/lib/apt/lists/*; \
+        if [ "$CRIS_IMG_ARCH" == 'amd64' ]; then \
```

Comment:
prefix _

Reply:
Do you mean that `[ "" == 'amd64' ]` may be true or an syntax error?

Reply:
It's not a syntax error because `[` is a command, e.g. `/usr/bin/[`.
But I've seen weird case that it doesn't handle empty arg well.

Reply:
As long as it gives you "false", I think it should be fine, isn't it?

Reply:
Let's make CRIS_IMG_ARCH guaranteed to be non-empty then.